src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,121 @@
+package duke;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import duke.exception.ArgumentNotValidException;
+import duke.exception.PromptCannotBeEmptyException;
+import duke.storage.StorageManager;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+import duke.utils.Input;
+import duke.utils.Output;
+
+public class Duke {
+    public static void main(String[] args) throws ArgumentNotValidException, PromptCannotBeEmptyException, IOException {
+        ArrayList<Task> tasks = StorageManager.readFileContents();
+        int indexTask;
+        Task newTask;
+
+        String[] parsedString;
+        String taskName = "";
+        String description;
+        String prompt;
+        String by;
+        String from;
+        String to;
+
+        Output.printIntroduction();
+
+        while (!taskName.equals("bye")) {
+            taskName = Input.scanWord();
+            try {
+                switch (taskName) {
+                case "list":
+                    Output.printTaskList(tasks);
+                    break;
+
+                case "todo":
+                    prompt = Input.scanPrompt("todo");
+                    if (prompt != null) {
+                        newTask = new Todo(prompt.trim());
+                        tasks.add(newTask);
+                        Output.printNewTaskMessage(newTask);
+                    }
+
+                    break;
+
+                case "deadline":
+                    prompt = Input.scanPrompt("deadline");
+                    if (prompt != null) {
+                        parsedString = Input.parseDeadline(prompt);
+                        if (parsedString != null) {
+                            description = parsedString[0];
```

Comment:
Try to avoid magic literals. There has been a few more instances in the rest of your code. You can use a named constant makes the code easier to understand because the name tells us more about the meaning of the number.